frontend/src/component/integrations/JiraIntegration/JiraImageContainer.tsx

```diff
@@ -0,0 +1,40 @@
+import { styled, Typography } from '@mui/material';
+
+import { formatAssetPath } from '../../../utils/formatPath';
+import { FC } from 'react';
+
+export const StyledContainer = styled('div')(({ theme }) => ({
+    display: 'flex',
+    gap: theme.spacing(2),
+    flexDirection: 'column',
+}));
+export const StyledTitle = styled(Typography)({
+    fontWeight: 'bold',
+});
+
+export const StyledImg = styled('img')({
+    maxWidth: '100%',
+    maxHeight: '100%',
+    width: 'auto',
+    height: 'auto',
+});
+
+interface JiraIntegrationProps {
+    title: string;
+    description: string;
+    src: string;
+}
+
+export const JiraImageContainer: FC<JiraIntegrationProps> = ({
+    title,
+    description,
+    src,
+}) => {
+    return (
+        <StyledContainer>
+            <StyledTitle>{title}</StyledTitle>
+            <Typography>{description}</Typography>
+            <StyledImg src={formatAssetPath(src)} />
+        </StyledContainer>
+    );
```

Comment:
What HTML elements is this rendered as (and what kind of a thing is it)? From the context, it sounds as if it might be an image with a caption? In that case, should we use the `figure` and `caption` elements? And should the `title` be an `h3` or something similar (depending on its position in the page)? If you wanna be very nitty gritty, the StyledContainer could probably be an `article` instead of a `div`, too, but that's not very important.

Also, the images don't have any captions or alt text, which is an accessibility issue. Based on the usage in the new code, I think the typography should be put in a `figcaption` component, linked to the images `figure` component.

Reply:
Done

Reply:
Did you push the changes? For some reason, I don't think I'm seeing them? 🤔